components/layout_2020/positioned.rs

```diff
@@ -42,6 +45,25 @@ pub(crate) enum AbsoluteBoxOffsets<NonStatic> {
 }
 
 impl AbsolutelyPositionedBox {
+    pub fn construct<'dom>(
+        context: &LayoutContext,
+        style: Arc<ComputedValues>,
+        display_inside: DisplayInside,
+        contents: Contents<impl NodeExt<'dom>>,
+    ) -> Self {
+        let request_content_sizes =
+            style.inline_size_is_auto() && !style.inline_box_offsets_are_both_auto();
+        Self {
+            contents: IndependentFormattingContext::construct(
+                context,
+                style,
+                display_inside,
+                contents,
+                request_content_sizes,
+            ),
+        }
+    }
+
```

Comment:
I feel like this commit would benefit from being split in two: first part with the introduction of the additional `construct` methods, second part with the introduction of the new `request_content_sizes` argument.

Reply:
I agree this would have been better, but at this point it’s not trivial to go back and make an intermediate commit that compiles. Do you feel strongly about this?